### PR TITLE
Adjust gateway account settlement settings

### PR DESCRIPTION
**Pull request by ashkarpetin**
## Summary
Add `immediately` method to settlement period
Add `anchor` to `date-interval` method
Reuse existing schemas

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Comment by ashkarpetin** [comment]
php please review

**Comment by ashkarpetin** [comment]
js please review

File: openapi/components/schemas/Scheduling/PeriodAnchor.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+type: object
+description: Instruction for calculating period anchor.
```
Comment (noon-dawg) [inline openapi/components/schemas/Scheduling/PeriodAnchor.yaml:2]: nitpick: I think adding 'the' makes this easier to read
```suggestion
description: Instruction for calculating the period anchor.
```

File: openapi/components/schemas/GatewayAccounts/FinancialSettings.yaml [hunk 2]
```diff
@@ -22,6 +22,8 @@ properties:
         bips:
           $ref: ../Bips.yaml
         period:
-          $ref: RiskReservePeriod.yaml
+          description: Instruction for the calculation of reserve release time.
```
Comment (noon-dawg) [inline openapi/components/schemas/GatewayAccounts/FinancialSettings.yaml:25]: nitpick: suggestion to use the same structure as the period anchor description 
```suggestion
          description: Instruction for calculating the reserve release time.
```

**Comment by ashkarpetin** [comment]
php please review

**Comment by ashkarpetin** [comment]
js please review

**Review by noon-dawg — APPROVED** [review]

**Review by nanobash — APPROVED** [review]

**Pull request merged**